packages/manager/modules/pci/src/projects/project/storages/databases/database.service.js

```diff
@@ -774,13 +774,12 @@ export default class DatabaseService {
     // pageSize,
     // page
   ) {
-    return this.iceberg(
-      `/cloud/project/${projectId}/database/${engine}/${databaseId}/queryStatistics`,
-    )
-      .query()
-      .expand('CachedObjectList-Pages')
-      .execute() // { limit: pageSize, offset: page }
-      .$promise.then(({ data: stats }) =>
+    return this.$http
+      .get(
+        `/cloud/project/${projectId}/database/${engine}/${databaseId}/queryStatistics`,
+        DatabaseService.getIcebergHeaders(),
```

Comment:
This might have to be reworked on right once we get pagination ?

Reply:
@varun257 Probably yes, but we will have to find a way to disable cache for iceberg calls made using iceberg service. Otherwise it would be difficult to implement pagination.